### PR TITLE
OKTA-454593 remove word wrap bold from simple text

### PR DESCRIPTION
**Pull request by praveendurai-okta**
## Description:

Remove the bold style in "your email" in text description .

## PR Checklist

- [X] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Screenshot/Video:
Screen Shot 2022-01-07 at 1 25 46 PM

Screen Shot 2022-01-07 at 1 26 14 PM

### Reviewers:

@mauriciocastillosilva-okta @indirathirumalaimurugan-okta 

### Issue:

- [OKTA-454593](https://oktainc.atlassian.net/browse/OKTA-454593)

### CODE REVIEW

**Review by indirathirumalaimurugan-okta — APPROVED** [review]
lgtm

File: packages/@okta/i18n/src/properties/login.properties [hunk 1]
```diff
@@ -926,10 +926,10 @@ oie.email.mfa.title = Verify with your email
 oie.email.verify.primaryButton = Send me an email
 # {0} is a user's email address
 oie.email.verify.subtitleWithEmailAddress = Verify with an email link or enter a code sent to <$1>{0}</$1>.
-oie.email.verify.subtitleWithoutEmailAddress = Verify with an email link or enter a code sent to <$1>your email</$1>.
+oie.email.verify.subtitleWithoutEmailAddress = Verify with an email link or enter a code sent to your email.
 # {0} is a user's email address
 oie.email.verify.alternate.magicLinkToEmailAddress = An email magic link was sent to <$1>{0}</$1>. 
-oie.email.verify.alternate.magicLinkToYourEmail = An email magic link was sent to <$1>your email</$1>. 
+oie.email.verify.alternate.magicLinkToYourEmail = An email magic link was sent to your email.
```
Comment (jmelberg-okta) [inline packages/@okta/i18n/src/properties/login.properties:932]: Please do not modify existing keys, as it'll produce L10N errors for non-English translations.

Instead, you will need to create a new key, and remove the stale key once translations are provided for the new key.

[More information here](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/741293778/Requesting+and+Receiving+Translations+i18n+and+l10n#Editing-strings) about this process.

**Review by jmelberg-okta — CHANGES_REQUESTED** [review]

File: packages/@okta/i18n/src/properties/login.properties (hunk 1 above)
Comment (praveendurai-okta) [reply packages/@okta/i18n/src/properties/login.properties:932]: @jmelberg-okta L10N errors will occur if we add new arguments and not while removing right? In this case we will see <$1> tag for non-english until translations is complete. we are targeting to merge this PR following the i18n release cadence. we were having a discussion about this here https://okta.slack.com/archives/CAZH2MWEL/p1641583735007400.

do you still think we need to have new key?

**Review by jmelberg-okta — APPROVED** [review]
Spoke offline - this change is intended for our monthly release and translations _should_ be available by EOM.

**Review by mauriciocastillosilva-okta — APPROVED** [review]
Make sure translations go in the same release

**Pull request closed without merging**